dbms/src/TiDB/Schema/TiDB.cpp

```diff
@@ -1264,6 +1264,8 @@ ColumnInfo toTiDBColumnInfo(const tipb::ColumnInfo & tipb_column_info)
     tidb_column_info.flag = tipb_column_info.flag();
     tidb_column_info.flen = tipb_column_info.columnlen();
     tidb_column_info.decimal = tipb_column_info.decimal();
+    tidb_column_info.origin_default_value = tipb_column_info.default_val();
```

Comment:
add comments about why we need to set `origin_default_value` but not `default_value`